sal_interfaces/Guider/Guider_Events.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>GuiderStatus_Stopped,GuiderStatus_Starting,GuiderStatus_Guiding,GuiderStatus_Error</Enumeration>
```

Comment:
That might be too narrow. Guider CCD needs to readout the image. Using GuiderStatus_Exposing,GuiderStatus_Readout might give more data about guiding progress. Not sure if that's needed, but might allow showing progress in GUI/EUI.

Reply:
Also what happens if you lost guider star(s)? Shall that not be reflected in some flags? Will then status switch to "waiting for guide * to re-appear"?

Reply:
The guider operates at a really high rate... I think if we where to capture the "status" with such fine details the state would be switching back and forth so fast it would be difficult to follow up. 

For the case where it loose the guide star, I think the best option would be to go to fault and then the status would be Error. If, internally, the Guider has the capability of waiting for a couple seconds before recovering the guide start, the status would remain "Guiding" and warning messages would log the condition. 

At the same time, if we ever feel we need a finer detail, we can update the enum set.